src/components/screen-reader-announcement/index.tsx

```diff
@@ -8,10 +8,28 @@ const ScreenReaderAnnouncement: React.FC<AnnouncementProps> = ({
 }: AnnouncementProps) => {
   useEffect(() => {
     if (message) {
-      AccessibilityInfo.announceForAccessibility(message);
+      setTimeout(
+        () => AccessibilityInfo.announceForAccessibility(message),
+        100,
+      );
```

Comment:
Shouldn't this component re-use the hook underneath since they pretty much do the same thing?